pkg/cloudprovider/providers/openstack/openstack_test.go

```diff
@@ -65,7 +65,7 @@ func WaitForVolumeStatus(t *testing.T, os *OpenStack, volumeName string, status
 			t.Logf("Volume (%s) status changed to %s after %v seconds\n",
 				volumeName,
 				status,
-				volumeStatusTimeoutSeconds)
+				time.Now().Second()-start)
```

Comment:
print the time taken since start. gives the wrong impression that that it took 30 seconds when it did not really

Reply:
nice catch